src/tests/plaintext.spec.js

```diff
@@ -3,85 +3,92 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { createEditor, serializePlainText } from './../EditorFactory';
-import spec from "./fixtures/spec"
-import xssFuzzVectors from './fixtures/xssFuzzVectors';
+import { createEditor, serializePlainText } from './../EditorFactory'
+import spec from './fixtures/spec'
+import xssFuzzVectors from './fixtures/xssFuzzVectors'
 
 const escapeHTML = (s) => {
-  return s.toString()
-      .split('&').join('&amp;')
-      .split('<').join('&lt;')
-      .split('>').join('&gt;')
-      .split('"').join('&quot;')
-      .split('\'').join('&#039;')
+	return s
+		.toString()
+		.split('&')
+		.join('&amp;')
+		.split('<')
+		.join('&lt;')
+		.split('>')
+		.join('&gt;')
+		.split('"')
+		.join('&quot;')
+		.split("'")
+		.join('&#039;')
```

Comment:
I prefer the previous version as these calls come im pairs of `split(..).join(...)` and that's easier to see in the original.

Normally I would agree that it's a good idea to put every function call on a new line... But here I don't. So I wonder if the rule that enforces this is really a good idea.